lib/web3/contract.js

```diff
@@ -176,6 +176,35 @@ var ContractFactory = function (eth, abi) {
      * @param {Function} callback
      * @returns {Contract} returns contract instance
      */
+    
+    
+      //   this.customByteCodeConvert
+
+  this.contractParmasConvertToBytes = function () {
```

Comment:
Yes this whole code is copy and paste , but there should be any function which gives bytecode for parameter of the contract new function . So i have introduce it.